README.md

```diff
@@ -379,6 +379,21 @@ The received webhook is available or not as boolean.
 1. Resolve with the received webhook is available or not.
 2. Reject if the input or output is not valid
 
+### getBranchList
+The parameters required are:
+
+| Parameter        | Type  | Required | Description |
+| :-------------   | :---- | :------- | :-------------|
+| config        | Object | Yes | Configuration Object |
+| config.scmUri | String | Yes | The scm uri (ex: `github.com:1234:branchName`) |
+| config.token | String | Yes | Access token for scm |
+
+#### Expected Outcome
+The array of object consists of branch name.
+
+#### Expected Promise Response
+1. Resolve with the array of object consists of branch name.
```

Comment:
Resolve with ~the~ `an` array of object`s` consist~s~`ing` of branch names.

Reply:
I will fix.